packages/divider/src/divider.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
+
+type DividerElement = React.ElementRef<"hr">;
+type RootProps = React.ComponentPropsWithoutRef<"hr">;
+
+export interface DividerProps extends RootProps {}
+
+export const Divider = React.forwardRef<DividerElement, Readonly<DividerProps>>(
+  ({ style, ...props }, forwardedRef) => (
+    <hr
+      ref={forwardedRef}
+      style={{
+        width: "100%",
+        border: "none",
+        borderTop: "1px solid #eaeaea",
+        margin: "26px 0",
```

Comment:
This margin shouldn't be on a component level, it should be on the template level only.

Reply:
Okay!